packages/webdriverio/src/commands/element/moveTo.js

```diff
@@ -14,7 +14,7 @@
 
 import { getElementRect, getScrollPosition } from '../../utils'
 
-export default async function moveTo (xoffset, yoffset) {
+export default async function moveTo (xoffset, yoffset, { releaseActions = true } = {}) {
```

Comment:
This optional flag would do no harm to existing code, but fixes the two-in-a-row `moveTo()` calls breakage on Safari.

Note that this is still not tested on Developer Preview.